src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    public static void doWork() {
+        System.out.println("Excavator started its work ");;
+
+    }
+
+    public static void stopWork() {
```

Comment:
Similar to the `doWork` method, `stopWork` should also be an instance method. Remove the `static` keyword to allow proper method overriding.